app/views/director/kids/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Edit <%= @kid.first_name %> <%= @kid.last_name %> 's Information </h1>
+
+<%= form_for @kid do |form| %>
+
+<%= form.label :first_name %>
+<%= form.text_field :first_name %>
+
+<%= form.label :last_name %>
+<%= form.text_field :last_name %>
+
+<%= form.label :birthdate %>
+<%= form.date_field :birthdate %>
+
+<%= form.label :social_insurance_number %>
```

Comment:
When I rendered the page, it wrote up "Social Insurance Number" and when I created a kid it wrote into the `sin` field of the Kid table. What defines what the key name should be is the line on bottom of this one. I can check to see if there's another way to do it though!